smg-automotive/components-pkg
# feat: rotate icon

References [subtask](https://autoricardo.atlassian.net/browse/DM-1480?atlOrigin=eyJpIjoiYjhiOTMwYTkwZmMxNGRlYzg3Nzc0ZGMzYzUwZGE5YTEiLCJwIjoiaiJ9)

## Motivation

needed an icon for reseting the filters

## How to test
Go to the [deployment](https://ilay-resetfilters-icon-components-pkg.branch.autoscout24.dev/?path=/docs/foundations-icons--docs)
Check if the icon is placed right or even if it is the right one
<img width="160" alt="Bildschirm­foto 2023-04-13 um 14 30 36" src="https://user-images.githubusercontent.com/110471838/231758789-92ed9fac-0858-4316-b206-10b5f360b6cd.png">
